polaris.shopify.com/src/components/Markdown/components/Colors/index.tsx

```diff
@@ -1,5 +1,7 @@
-import * as colorsObj from '../../../../../../polaris-tokens/dist/esm/src/colors.mjs';
+import {Grid} from '@shopify/polaris';
```

Comment:
<img width="2032" alt="Screenshot 2023-09-25 at 1 35 16 PM" src="https://github.com/Shopify/polaris/assets/4642404/30bf2a6f-7e46-4e5f-a67a-e401de7cbcc7">
seeing this error when clicking on the color palette